src/controllers/scripting/legacy/controllerscriptenginelegacy.h

```diff
@@ -125,7 +125,7 @@ class ControllerScriptEngineLegacy : public ControllerScriptEngineBase {
     std::unordered_map<QString, std::unique_ptr<mixxx::qml::QmlMixxxControllerScreen>> m_rootItems;
     QList<LegacyControllerMapping::QMLModuleInfo> m_modules;
     QList<LegacyControllerMapping::ScreenInfo> m_infoScreens;
-    QString m_resourcePath{QStringLiteral(".")};
```

Comment:
How do you feel about keeping this default till we implement the proper module publishing?

Reply:
Here we have a different topic. I have removed the default . because is never used. It is set via setResourcePath() to the correct value. Appart from that . would require to start Mixxx from the res directory

Reply:
Yeah, actually this should not break the current behaviour, but I thought it would be best to default the QML path to be `./qml` rather than `/qml`, but since we are calling `setResourcePath` before loading the scene, I'm happy to ignore that